nion/instrumentation/MultiAcquire.py

```diff
@@ -70,96 +82,329 @@ def copy(self):
         return self.__copy__()
 
 
+class ScanDataChannel(scan_base.SynchronizedDataChannelInterface):
```

Comment:
I don't think this should be derived from scan_base.SynchronizedDataChannelInterface, which is for cameras. The 'update' method conflicts in my type checker.